charts/splunk-connect-for-snmp/Chart.lock

```diff
@@ -7,6 +7,6 @@ dependencies:
   version: 17.3.18
 - name: mibserver
   repository: https://pysnmp.github.io/mibs/charts/
-  version: 1.15.1
-digest: sha256:9e01a49851f1ea1ba6b1c75785ac2b9a03ab5142588ef8a071f694d904c0c1f1
-generated: "2023-02-09T15:52:42.216766+01:00"
+  version: 1.15.2
```

Comment:
maybe let's not override the helm chart version here, would be good to open another PR specifically for this purpose